src/Stryker.Core/Stryker.Core/Reporters/HtmlReporter/ProcessWrapper/ProcessWrapper.cs

```diff
@@ -1,14 +1,15 @@
 using System.Diagnostics;
+using System.Runtime.InteropServices;
 
 namespace Stryker.Core.Reporters.HtmlReporter.ProcessWrapper
 {
     public class ProcessWrapper : IProcessWrapper
```

Comment:
Perhaps we should rename this to something like `ReportOpener` since ProcessWrapper is pretty technical for what it actually does.

Reply:
Nice!